strapi/strapi
# Protect telemetry properties route

### What does it do?

Makes admin/telemetry-properties route private just for admin users and modifies the frontend so that it doesn't break because of this backend protection.

### Why is it needed?

To avoid potential security gaps exposing sensitive data of the strapi instance. 

### How to test it?

Try to go to admin/telemetry-properties and check that it gives a 401 error if a valid Bearer token is not provided. Check also that in the admin interface, the admin/telemetry-properties route is not being called when the frontend doesn't have a valid token.

### Related issue(s)/PR(s)

Needs to be merged before: https://github.com/strapi/strapi/pull/14900
